src/main/java/ezschedule/model/event/UniqueEventList.java

```diff
@@ -50,9 +50,9 @@ public void add(Event toAdd) {
     }
 
     /**
-     * Replaces the person {@code target} in the list with {@code editedPerson}.
+     * Replaces the event {@code target} in the list with {@code editedPerson}.
```

Comment:
`{@code editPerson}` should be changed too!

Reply:
Fixed